cumulus/parachains/runtimes/assets/asset-hub-westend/src/bridge_to_ethereum_config.rs

```diff
@@ -121,11 +196,18 @@ impl snowbridge_pallet_system_frontend::Config for Runtime {
 	type XcmSender = XcmRouter;
 	#[cfg(feature = "runtime-benchmarks")]
 	type XcmSender = DoNothingRouter;
+	#[cfg(not(feature = "runtime-benchmarks"))]
 	type AssetTransactor = AssetTransactors;
+	#[cfg(feature = "runtime-benchmarks")]
+	type AssetTransactor = SuccessfulTransactor;
 	type EthereumLocation = FeeAsset;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 	type BridgeHubLocation = BridgeHubLocation;
 	type UniversalLocation = UniversalLocation;
 	type PalletLocation = SystemFrontendPalletLocation;
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type Swap = AssetConversion;
+	#[cfg(feature = "runtime-benchmarks")]
+	type Swap = DoNothingSwap;
```

Comment:
why are you ignoring the weight of transacting or swapping assets when benchmarking `snowbridge_pallet_system_frontend`?

Reply:
Fixed the benchmarks over here: https://github.com/paritytech/polkadot-sdk/pull/8271/commits/e70e1064ff47a98d07f6bde79205a0665b70b751